transport/src/test/java/io/scalecube/transport/TransportTest.java

```diff
@@ -88,6 +90,36 @@ public void testInvalidListenAddress() {
   }
 
   @Test
+  public void testPortAutoIncrementRaceConditions() throws Exception {
```

Comment:
maybe should consider to add a test that checks bind exception when port auto increment is false?